Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -280,308 +104,260 @@ def parse_reputation_human_readable(reputation):
     return res
 
 
-def parse_reference_human_readable(reference):
-    agent_guid = reference.get(FirstRefProp.SYSTEM_GUID)
-    date = reference.get(FirstRefProp.DATE)
+def parse_reference(reference):
+    agent_guid = reference.get('agentGuid')
+    date = reference.get('date')
+    try:
+        date = datetime.fromtimestamp(date)
+    except ValueError:
+        date = datetime.fromtimestamp(date / 1000)
 
     return {
-        'Date': epoch_to_localtime(date),
+        'Date': str(date),
         'AgentGuid': agent_guid.replace('{', '').replace('}', '')  # remove brackets if exist
     }
 
 
-def reputations_to_human_readable(reputations):
-    return [parse_reputation_human_readable(rep) for rep in reputations]
+def reputations_to_table(reputations):
+    return [parse_reputation(rep) for rep in reputations]
 
 
-def references_to_human_readable(references):
-    return [parse_reference_human_readable(ref) for ref in references]
+def references_to_table(references):
+    return [parse_reference(ref) for ref in references]
 
 
 def trust_level_to_score(trust_level):
-    if (trust_level >= TrustLevel.MIGHT_BE_TRUSTED):
-        return Common.DBotScore.GOOD
+    if (trust_level >= 70):
+        return 1
+    elif (trust_level == 30):
+        return 2
+    elif (trust_level == 0 or trust_level == 50):
+        return 0
+    elif (trust_level < 30):
+        return 3
+    else:
+        # Shouldn't reach here, as the API doesn't support 31-69 values except for 50)
+        return 0
```

Comment:
wrong logic, if the value can be 60, what will happen?
If it cannot, please document that in the function docs.